.github/workflows/node.yml

```
@@ -0,0 +1,29 @@
+jobs:
+  build:
+    env:
+      CI: true
+      MONGO_URL: mongodb://localhost/nasa
+    strategy:
+      matrix:
+        node-version: [14.x, 16.x]
+        mongodb-version: ["4.4"]
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
```

actionlint

the runner of "actions/checkout@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)